bigtable/spark/src/test/scala/example/DataFrameDemoLuncher.scala

```diff
@@ -0,0 +1,7 @@
+package example
+
+object DataFrameDemoLuncher extends App {
```

Comment:
typo in 'Launcher'

Reply:
Time to rewrite it to be a full-blown test, isn't it? I'm adding a testing framework to the project. In next commit...

Reply:
SGTM - if presubmit checks cannot run the test for now, we will manually run and verify so we can merge in

Reply:
Added a simple test that gets rid of this launcher. Considered resolved.